model/enduser/deprecated/registry-deprecated.yaml

```
@@ -2,23 +2,17 @@ groups:
   - id: registry.enduser.deprecated
     type: attribute_group
     display_name: Deprecated End User Attributes
-    brief: Describes deprecated enduser attributes. Complete enduser namespace has been deprecated
+    brief: "Describes deprecated enduser attributes."
     attributes:
-      - id: enduser.id
-        type: string
-        stability: experimental
-        deprecated: Replaced by `user.id` attribute.
-        brief: "Deprecated, use `user.id` instead."
-        examples: 'username'
       - id: enduser.role
         type: string
+        deprecated: "Removed."
```

Any reason to change this one?

```suggestion
        deprecated: Replaced by `user.roles` attribute.
```

---

i can revert it.